application/grifts/db.go

```diff
@@ -21,16 +21,9 @@ var _ = grift.Namespace("db", func() {
 		}
 
 		// ORGANIZATIONS Table
-		//organizationUUID1, _ := uuid.FromString("f3a79b30-f00e-48a0-a64d-e27748dea22d")
-		organizationUUID2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
+		organizationUuid2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
```

Comment:
Did this get changed by something automated?  In any case, it doesn't match line 29 or the all-caps "rule".

Reply:
Well, it's more of a "guideline"

Reply:
```suggestion
		organizationUUID2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
```

Reply:
I got reverted because I missed it in resolving a merge conflict.